addons/project/static/src/views/project_task_kanban/project_task_kanban_renderer.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates>
+    <t t-name="project.ProjectTaskKanbanRenderer" t-inherit="web.KanbanRenderer" t-inherit-mode="primary" owl="1">
+        <KanbanColumnQuickCreate position="replace">
+            <ProjectTaskKanbanColumnQuickCreate
+                folded="state.columnQuickCreateIsFolded"
+                onFoldChange="folded => state.columnQuickCreateIsFolded = props.list.groups.length > 0 and folded"
+                onValidate="(name, fold, model) => props.list.createGroup(name, fold, model)"
```

Comment:
I think this is a generic claim. So it can be done in web.
And it should be possible without the model :thinking:

Reply:
i didn't find the information in https://github.com/odoo/odoo/pull/107633/files#:~:text=()%20%7B-,async%20createGroup()%20%7B,if%20(this.isGroupedByPersonalStages)%20%7B,-if%20(this
that's why i passed it from `applyExamples()`

Yes it can be generic and useful in all kanban examples but i prefered to respect the scope of the task and avoid breaking something on web.

Reply:
My point is: in my opinion, this legitimately goes beyond our scope.
About finding the model : in createGroup, this will do it `this.model.root.fields[this.model.root.groupBy].relation` :slightly_smiling_face:

Reply:
i will keep the current behavior by sending modal field from applyExamples as i want it to be applied only when applying kanban examples. the method is also called when adding a stage from `all tasks`, `my tasks`, ` project tasks grouped by any other field` so it will lead to many wrong corner cases like grouping by assignees or any field != stage.type

Reply:
@RaoufGhrissi Without sending a message to the team (in direct message for instance), we will never know you need another opinion for that. 🙂 

I agree with @auon-odoo if we can have the information without passing it in argument then we should use it instead of adding new parameter. 🙂 

I also agree to alter the generic code in `web` to be able to have your changes for Helpdesk app for instance. 🙂

Reply:
hello @xavierbol  i just fixed some logique, edited my comment and "grey"ed the task now. it's a new comment and i was planning to text you.
Okay, i will try to move it to web, but keep in mind that we still need a param to know if we are in kanban examples context or not ( you can check the updated code )

Reply:
*but keep in mind that we still need a param to know if we are in kanban examples context or not ( you can check the updated code )*
I don't understand why we need that parameter. For me, if we are in `applyExamples` method then we are in kanban examples context.

May I miss something?

Reply:
hello @xavierbol , the method `createGroup` where i made my changes is called when we apply kanban examples and when we quick create manually grouping by anything and in `all tasks`, `my tasks`, `project tasks`, i want my changes to only be applied when i'm in `project tasks` applying the kanban examples grouped by `"project.task.type"`